include/spdlog/details/file_helper-inl.h

```diff
@@ -118,7 +118,7 @@ SPDLOG_INLINE std::tuple<filename_t, filename_t> file_helper::split_by_extension
     }
 
     // treat cases like "/etc/rc.d/somelogfile or "/abc/.hiddenfile"
-    auto folder_index = fname.rfind(details::os::folder_sep);
+    auto folder_index = fname.find_last_of(details::os::folder_seps_filename);
```

Comment:
Will  break if ```folder_seps_filename``` is more than single  char. Let's use the original ```rfind()```

Reply:
That's the intention: each character in `folder_seps_filename` is considered as a possible folder separator. `find_last_of` searches for any character in `folder_seps_filename` instead of searching for the entirety of the string.